lib/repeatable/expression/difference.rb

```diff
@@ -0,0 +1,26 @@
+module Repeatable
+  module Expression
+    class Difference < Base
+      def initialize(included:, excluded:)
+        @included = included
+        @excluded = excluded
+      end
+
+      attr_reader :included, :excluded
```

Comment:
Would you mind putting these `attr_reader`s in a `protected` section so they can be used by `Difference#==` but can't be accessed by other classes?  That's how this sort of thing is currently handled in [`Set`](https://github.com/molawson/repeatable/blob/589fe22f9aec460314e704e1bc91434502f430ad/lib/repeatable/expression/set.rb#L23-L25)